From 2f9a057be074726bca67ee4a1028b914ba1818a8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mikko=20Salom=C3=A4ki?= <ms@datarespons.se>
Date: Mon, 18 May 2020 20:51:30 +0000
Subject: [PATCH] Add-vdt6010-to-mx6-Kconfig

---
 arch/arm/mach-imx/mx6/Kconfig | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/mach-imx/mx6/Kconfig b/arch/arm/mach-imx/mx6/Kconfig
index f9f576d403..3dd8095ca1 100644
--- a/arch/arm/mach-imx/mx6/Kconfig
+++ b/arch/arm/mach-imx/mx6/Kconfig
@@ -670,6 +670,15 @@ config TARGET_BRPPT2
           Support
           B&R BRPPT2 platform
           based on Freescale's iMX6 SoC
+          
+config TARGET_VDT6010
+	bool "vdt6010"
+	select MX6QDL
+	select BOARD_LATE_INIT
+	select SUPPORT_SPL
+	select DM
+	select DM_THERMAL
+	select BOARD_EARLY_INIT_F
 endchoice
 
 config SYS_SOC
@@ -729,5 +738,6 @@ source "board/udoo/neo/Kconfig"
 source "board/wandboard/Kconfig"
 source "board/warp/Kconfig"
 source "board/BuR/brppt2/Kconfig"
+source "board/datarespons/vdt6010/Kconfig"
 
 endif
