From f4a347a69941405e791c9e442f1c0caf4e53688c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mikko=20Salom=C3=A4ki?= <ms@datarespons.se>
Date: Mon, 18 May 2020 20:51:00 +0000
Subject: [PATCH] Add-vdt6010-to-dts-makefile

---
 arch/arm/dts/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index 820ee9733a..8782b720a7 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -658,6 +658,7 @@ dtb-y += \
 	imx6qp-sabreauto.dtb \
 	imx6qp-sabresd.dtb \
 	imx6qp-wandboard-revd1.dtb \
+	vdt6010-q-revA.dtb \
 
 endif
 
