From b7e62ee00cb5ab0df3b435c164a64666ca9135d9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mikko=20Salom=C3=A4ki?= <ms@datarespons.se>
Date: Wed, 15 May 2019 13:44:39 +0200
Subject: [PATCH] Set backlight PWM freq to 1KHz

---
 board/datarespons/vdt6010/vdt6010.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/board/datarespons/vdt6010/vdt6010.c b/board/datarespons/vdt6010/vdt6010.c
index 9906c10e3a..226f60053b 100644
--- a/board/datarespons/vdt6010/vdt6010.c
+++ b/board/datarespons/vdt6010/vdt6010.c
@@ -45,6 +45,9 @@ DECLARE_GLOBAL_DATA_PTR;
 #include "vdt6010_pins.h"
 #include "vdt6010_gpio.h"
 
+#define BACKLIGHT_FREQ_HZ 1000
+#define BACKLIGHT_PERIOD_NS 1000000000 / BACKLIGHT_FREQ_HZ
+
 /*
  *
  * board_early_init_f
@@ -58,7 +61,7 @@ static void setup_backlight()
 	gpio_set_value(LED_VCC_EN, 1);
 	gpio_set_value(LED_EN, 1);
 
-	if (pwm_config(0, 100000, 100000)) {
+	if (pwm_config(0, BACKLIGHT_PERIOD_NS, BACKLIGHT_PERIOD_NS)) {
 		printf("%s: pwm_config failed\n", __func__);
 	}
 
