From a9854be399da19fb3daf2e21ab415d8a2a2b8fcf Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mikko=20Salom=C3=A4ki?= <ms@datarespons.se>
Date: Wed, 29 May 2019 15:55:29 +0200
Subject: [PATCH] Remove double-enabled kernel append console

---
 include/configs/vdt6010.h | 16 ++++++----------
 1 file changed, 6 insertions(+), 10 deletions(-)

diff --git a/include/configs/vdt6010.h b/include/configs/vdt6010.h
index ecb40dcb17..dd0f3b2b8f 100644
--- a/include/configs/vdt6010.h
+++ b/include/configs/vdt6010.h
@@ -34,7 +34,7 @@
 #define CONFIG_SF_DEFAULT_CS		0
 #define CONFIG_SF_DEFAULT_SPEED		20000000
 #define CONFIG_SF_DEFAULT_MODE		SPI_MODE_0
-#define CONFIG_SPI_MEM /* Required for SPL build as of 2019.11 */
+#define CONFIG_SPI_MEM /* Required for SPL build as of 2019.04 */
 
 /* Flash and environment organization */
 /*#define CONFIG_SYS_NO_FLASH*/
@@ -54,7 +54,6 @@
 #define CONFIG_MXC_UART_BASE	UART5_BASE
 #define CONSOLE_DEV		"ttymxc4"
 #define CONFIG_IMX_WATCHDOG
-#undef CONFIG_IMX_WATCHDOG_USE_WD2
 
 #define CONFIG_MMCROOT			"/dev/mmcblk0p1"
 
@@ -198,7 +197,7 @@
 	"initrd_file=/boot/initrd\0" \
 	"loglevel="xstr(CONFIG_LOGLEVEL)"\0" \
 	"consoleblank=0\0" \
-	"showtty=console=ttymxc0,115200 console=tty1\0" \
+	"showtty=console=tty1\0" \
 	"setargs=setenv bootargs console=${console} root=${rootdev} rootwait ro rootfstype=ext4 consoleblank=${consoleblank} loglevel=${loglevel} ${showtty}\0" \
 	"bootdev=0\0" \
 	"bootpart=1\0" \
@@ -301,15 +300,12 @@
 
 #define CMD_CRC32
 
-#ifdef CONFIG_CMD_PCI
+/* PCIe configs */
+/*
 #define CONFIG_PCI_SCAN_SHOW
+#define CONFIG_PCIE_IMX_PERST_GPIO	IMX_GPIO_NR(4, 8)
 #define CONFIG_PCIE_IMX
-#if 0
-#define CONFIG_PCIE_IMX_PERST_GPIO	IMX_GPIO_NR(7, 12)
-#define CONFIG_PCIE_IMX_POWER_GPIO	IMX_GPIO_NR(3, 19)
-#endif
-#endif
-
+*/
 
 /* I2C Configs */
 #define CONFIG_SYS_I2C
