From a71c9930188de8a554bc06833c0b6766a00a04b5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mikko=20Salom=C3=A4ki?= <ms@datarespons.se>
Date: Tue, 11 Jun 2019 13:10:53 +0200
Subject: [PATCH] Adjust DDR IOMUXC_SW_PAD_CTL

---
 board/datarespons/vdt6010/spl.c | 66 ++++++++++++++++++++---------------------
 1 file changed, 33 insertions(+), 33 deletions(-)

diff --git a/board/datarespons/vdt6010/spl.c b/board/datarespons/vdt6010/spl.c
index b725ef6861..a7f2785ce4 100644
--- a/board/datarespons/vdt6010/spl.c
+++ b/board/datarespons/vdt6010/spl.c
@@ -117,49 +117,49 @@ int board_mmc_init(bd_t *bis)
 #define MX6DQ_INPUT_DIFFERENTIAL 0x00020000
 
 static const struct mx6dq_iomux_ddr_regs mx6dq_ddr_ioregs = {
-	.dram_sdclk_0 = MX6DQ_DS_48_OHM,
-	.dram_sdclk_1 = MX6DQ_DS_48_OHM,
-	.dram_cas = MX6DQ_DS_48_OHM,
-	.dram_ras = MX6DQ_DS_48_OHM,
-	.dram_reset = MX6DQ_DS_48_OHM,
+	.dram_sdclk_0 = MX6DQ_INPUT_DIFFERENTIAL | MX6DQ_DS_40_OHM,
+	.dram_sdclk_1 = MX6DQ_INPUT_DIFFERENTIAL | MX6DQ_DS_40_OHM,
+	.dram_cas = MX6DQ_INPUT_DIFFERENTIAL | MX6DQ_DS_40_OHM,
+	.dram_ras = MX6DQ_INPUT_DIFFERENTIAL | MX6DQ_DS_40_OHM,
+	.dram_reset = MX6DQ_INPUT_DIFFERENTIAL | MX6DQ_DS_40_OHM,
 	.dram_sdba2 = MX6DQ_DS_GRP,
 	.dram_sdcke0 = 0x00003000, // reserved bits
 	.dram_sdcke1 = 0x00003000, // reserved bits
-	.dram_sdodt0 = MX6DQ_DS_48_OHM,
-	.dram_sdodt1 = MX6DQ_DS_48_OHM,
-	.dram_sdqs0 = MX6DQ_DS_48_OHM,
-	.dram_sdqs1 = MX6DQ_DS_48_OHM,
-	.dram_sdqs2 = MX6DQ_DS_48_OHM,
-	.dram_sdqs3 = MX6DQ_DS_48_OHM,
-	.dram_sdqs4 = MX6DQ_DS_48_OHM,
-	.dram_sdqs5 = MX6DQ_DS_48_OHM,
-	.dram_sdqs6 = MX6DQ_DS_48_OHM,
-	.dram_sdqs7 = MX6DQ_DS_48_OHM,
-	.dram_dqm0 = MX6DQ_DS_48_OHM,
-	.dram_dqm1 = MX6DQ_DS_48_OHM,
-	.dram_dqm2 = MX6DQ_DS_48_OHM,
-	.dram_dqm3 = MX6DQ_DS_48_OHM,
-	.dram_dqm4 = MX6DQ_DS_48_OHM,
-	.dram_dqm5 = MX6DQ_DS_48_OHM,
-	.dram_dqm6 = MX6DQ_DS_48_OHM,
-	.dram_dqm7 = MX6DQ_DS_48_OHM,
+	.dram_sdodt0 = 0x00003000 | MX6DQ_DS_40_OHM,
+	.dram_sdodt1 = 0x00003000 | MX6DQ_DS_40_OHM,
+	.dram_sdqs0 = MX6DQ_DS_40_OHM,
+	.dram_sdqs1 = MX6DQ_DS_40_OHM,
+	.dram_sdqs2 = MX6DQ_DS_40_OHM,
+	.dram_sdqs3 = MX6DQ_DS_40_OHM,
+	.dram_sdqs4 = MX6DQ_DS_40_OHM,
+	.dram_sdqs5 = MX6DQ_DS_40_OHM,
+	.dram_sdqs6 = MX6DQ_DS_40_OHM,
+	.dram_sdqs7 = MX6DQ_DS_40_OHM,
+	.dram_dqm0 = MX6DQ_INPUT_DIFFERENTIAL | MX6DQ_DS_40_OHM,
+	.dram_dqm1 = MX6DQ_INPUT_DIFFERENTIAL | MX6DQ_DS_40_OHM,
+	.dram_dqm2 = MX6DQ_INPUT_DIFFERENTIAL | MX6DQ_DS_40_OHM,
+	.dram_dqm3 = MX6DQ_INPUT_DIFFERENTIAL | MX6DQ_DS_40_OHM,
+	.dram_dqm4 = MX6DQ_INPUT_DIFFERENTIAL | MX6DQ_DS_40_OHM,
+	.dram_dqm5 = MX6DQ_INPUT_DIFFERENTIAL | MX6DQ_DS_40_OHM,
+	.dram_dqm6 = MX6DQ_INPUT_DIFFERENTIAL | MX6DQ_DS_40_OHM,
+	.dram_dqm7 = MX6DQ_INPUT_DIFFERENTIAL | MX6DQ_DS_40_OHM,
 };
 
 static const struct mx6dq_iomux_grp_regs mx6dq_grp_ioregs = {
 	.grp_ddr_type = MX6DQ_DDR_TYPE_DD3,
 	.grp_ddrpke = MX6DQ_DDRPKE_DISABLED,
-	.grp_addds = MX6DQ_DS_48_OHM,
-	.grp_ctlds = MX6DQ_DS_48_OHM,
+	.grp_addds = MX6DQ_DS_40_OHM,
+	.grp_ctlds = MX6DQ_DS_40_OHM,
 	.grp_ddrmode_ctl = MX6DQ_INPUT_DIFFERENTIAL,
 	.grp_ddrmode = MX6DQ_INPUT_DIFFERENTIAL,
-	.grp_b0ds = MX6DQ_DS_48_OHM,
-	.grp_b1ds = MX6DQ_DS_48_OHM,
-	.grp_b2ds = MX6DQ_DS_48_OHM,
-	.grp_b3ds = MX6DQ_DS_48_OHM,
-	.grp_b4ds = MX6DQ_DS_48_OHM,
-	.grp_b5ds = MX6DQ_DS_48_OHM,
-	.grp_b6ds = MX6DQ_DS_48_OHM,
-	.grp_b7ds = MX6DQ_DS_48_OHM,
+	.grp_b0ds = MX6DQ_DS_40_OHM,
+	.grp_b1ds = MX6DQ_DS_40_OHM,
+	.grp_b2ds = MX6DQ_DS_40_OHM,
+	.grp_b3ds = MX6DQ_DS_40_OHM,
+	.grp_b4ds = MX6DQ_DS_40_OHM,
+	.grp_b5ds = MX6DQ_DS_40_OHM,
+	.grp_b6ds = MX6DQ_DS_40_OHM,
+	.grp_b7ds = MX6DQ_DS_40_OHM,
 };
 
 static const struct mx6_mmdc_calibration mx6dq_calib_NT5CC256M16EP_EK = {
